### PR TITLE
fix: bump @synonymdev/react-native-keychain to v9.1.1

### LINKED ISSUE #1916: [Bug]: Hot reloading on Android broken
### Describe the bug

https://synonymworkspace.slack.com/archives/C01H106JRQB/p1714631786516419

Development issue only, low prio

### Operating system

Android

### Bitkit version

04959b72

## Preview
Image

### PR DESCRIPTION
**Pull request by limpbrains**
### Description

Update to this fork https://github.com/synonymdev/react-native-keychain/pull/2

### Linked Issues/Tasks

closes #1916 #2332 

### Type of change

Bug fix

### Tests

No test

### CODE REVIEW

**Comment by pwltr** [comment]
Needs Podfile update?

**Comment by limpbrains** [comment]
Updated

**Review by pwltr — APPROVED** [review]

**Comment by limpbrains** [comment]
@ovitrif 
Locally, it works fine, but on GitHub Actions, the seed never loads, even after 30 seconds. 
Additionally, there are no errors in the application log. 
Here's an example of the text run:

https://github.com/user-attachments/assets/c00e256f-7998-4eff-8995-2e10d8e53f84

File: src/screens/Settings/Backup/ShowMnemonic.tsx [hunk 1]
```diff
@@ -44,25 +44,37 @@ const ShowMnemonic = ({
...
-			setSeed(res.value.split(' '));
-		});
-		getBip39Passphrase().then(setPassphrase);
+
+			setSeed(mnemoncicResult.value.split(' '));
+			setPassphrase(bip39PassphraseResult);
+		};
+
+		getSeed();
 	}, [t]);
```
Comment (limpbrains) [inline src/screens/Settings/Backup/ShowMnemonic.tsx:66]: @ovitrif 
I've fixed the issue that, I was complaining about earlier, by removing concurrent requests for the keychain.
It is ok, but I think there is a bug in keychain library, maybe you are intrested

**Pull request merged**